examples/features/mutual_tls/client/main.go

```diff
@@ -0,0 +1,78 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// The client demonstrates how to supply an OAuth2 token for every RPC.
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/examples/data"
+	ecpb "google.golang.org/grpc/examples/features/proto/echo"
+)
+
+var addr = flag.String("addr", "localhost:50051", "the address to connect to")
+
+func callUnaryEcho(client ecpb.EchoClient, message string) {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	resp, err := client.UnaryEcho(ctx, &ecpb.EchoRequest{Message: message})
+	if err != nil {
+		log.Fatalf("client.UnaryEcho(_) = _, %v: ", err)
+	}
+	fmt.Println("UnaryEcho: ", resp.Message)
+}
+
+func main() {
+	flag.Parse()
+
+	clientCredential, err := tls.LoadX509KeyPair(
+		data.Path("x509/client_cert.pem"),
+		data.Path("x509/client_key.pem"),
+	)
+	if err != nil {
+		log.Fatalf("failed to load client cert: %v", err)
+	}
+
+	trustedServerCAs, err := data.NewCertPool(data.Path("x509/ca_cert.pem"))
+	if err != nil {
+		log.Fatalf("failed to load CA cert: %v", err)
+	}
+
+	tlsConfig := &tls.Config{
+		ServerName:   "x.test.example.com",
+		Certificates: []tls.Certificate{clientCredential},
+		RootCAs:      trustedServerCAs,
+	}
+
+	conn, err := grpc.Dial(*addr, grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)))
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer conn.Close()
+	rgc := ecpb.NewEchoClient(conn)
```

Comment:
What does `rgc` mean? Prefer inlining this into the call to `callUnaryEcho`.

Reply:
This was a copy/paste from the one of the other examples. I have inlined this.